src/gsy_e/gsy_e_core/rq_job_handler.py

```diff
@@ -26,19 +26,21 @@ def decompress_and_decode_queued_strings(queued_string: bytes) -> Dict:
 
 def launch_simulation_from_rq_job(scenario: bytes, settings: Optional[Dict],
                                   events: Optional[str], aggregator_device_mapping: str,
-                                  saved_state: bytes, job_id: str):
+                                  saved_state: bytes, job_id: str,
+                                  connect_to_profiles_db: bool = True):
     # pylint: disable=too-many-arguments, too-many-locals
     """Launch simulation from rq job."""
     logging.getLogger().setLevel(logging.ERROR)
     scenario = decompress_and_decode_queued_strings(scenario)
-    gsy_e.constants.CONFIGURATION_ID = scenario.pop("configuration_uuid")
-    if "collaboration_uuid" in scenario:
-        gsy_e.constants.EXTERNAL_CONNECTION_WEB = True
-        GlobalConfig.IS_CANARY_NETWORK = scenario.pop("is_canary_network", False)
-        gsy_e.constants.RUN_IN_REALTIME = GlobalConfig.IS_CANARY_NETWORK
+    if isinstance(scenario, dict):
+        gsy_e.constants.CONFIGURATION_ID = scenario.pop("configuration_uuid")
+        if "collaboration_uuid" in scenario:
+            gsy_e.constants.EXTERNAL_CONNECTION_WEB = True
+            GlobalConfig.IS_CANARY_NETWORK = scenario.pop("is_canary_network", False)
+            gsy_e.constants.RUN_IN_REALTIME = GlobalConfig.IS_CANARY_NETWORK
     saved_state = decompress_and_decode_queued_strings(saved_state)
-    log.error("Starting simulation with job_id: %s and configuration id: %s",
-              job_id, gsy_e.constants.CONFIGURATION_ID)
+    log.info("Starting simulation with job_id: %s and configuration id: %s",
```

Comment:
This will clatter our logs on the simulation pod. I remember explicitly changing the log level to error.